app/views/devise/confirmations/new.html.slim

```diff
@@ -1,10 +1,9 @@
 h2 Resend confirmation instructions
 = form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f|
   = devise_error_messages!
-  .field
+  .form-group
     = f.label :email
-    br/
-    = f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email)
-  .actions
-    = f.submit "Resend confirmation instructions"
+    = f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email), class: 'form-control'
+  .form-group
+    = f.submit "Resend confirmation instructions", class: 'btn btn-primary'
```

Comment:
@marderer all plain text should be moved to locales in order to have ability to switch to other languages later